src/Xml/XmlRuntime.fs

```diff
@@ -57,7 +57,6 @@ open System.Xml.Linq
 #nowarn "10001"
 
 /// Underlying representation of types generated by XmlProvider
-[<StructuredFormatDisplay("{_Print}")>]
```

Comment:
Revert this line

Reply:
I've reverted the removal of that line.  Do you want me to remove the line itself?